feature/interface/singleton/otg_tests/singleton_test/README.md

```diff
@@ -73,6 +73,37 @@ a new testbed configuration with the desired port types.
 *   Ensure inbound and outbound unicast counters are the same
 *   Ensure counters increment at the selected SAMPLE interval
 
+### RT-5.1.4 [TODO: https://github.com/openconfig/featureprofiles/issues/2338]
+#### Breakout must be explicitly configured by gNMI client
+
+*   On DUT Port-1 with a QSFP-DD 400GBASE-DR4 transceiver inserted
+*   Ensure no breakout is configured
+*   Set Port-1 port-speed to 100G
+    *   /interfaces/interface/ethernet/config/port-speed
+*   Validate that the DUT does not create breakouts implicitly and does not set the breakout speed
+    *   /components/component/port/breakout-mode/groups/group/config
+    *   /components/component/port/breakout-mode/groups/group/config/index
+    *   /components/component/port/breakout-mode/groups/group/config/breakout-speed
+*   Validate the port state changes to "DOWN"
+    *   /interfaces/interface/state/oper-status
+
+### RT-5.1.5 [TODO: https://github.com/openconfig/featureprofiles/issues/2338]
+#### Setting port-speed on interface that have breakout configured should not be allowed
+
+*   Configure a breakout on Port-1 to 4x100 Gig
+    *   /components/component/port/breakout-mode/groups/group/config
+*   Try to set port speed of Port-1 to 100G
+*   Validate the port-speed is rejected
```

Comment:
Might be good to add some explanation for this:   "Since a breakout port is not expected to support `port-speed`, verify the gnmi Set operation is rejected.  "

Reply:
updated